python-sdk/conftest.py

```diff
@@ -132,6 +132,9 @@ def database_table_fixture(request):
     conn_id = DATABASE_NAME_TO_CONN_ID[database_name]
     database = create_database(conn_id)
     table = params.get("table", Table(conn_id=database.conn_id, metadata=database.default_metadata))
+    if not isinstance(table, TempTable):
+        # We add a unique suffix to the table name to make the name unique across runs
+        table.name += create_unique_table_name(UNIQUE_HASH_SIZE)
```

Comment:
@feluelle do we account for an upper limit on table name length?

Reply:
No, do we have one? 🤔 I am not aware of any validation on our side. We could probably at some point add one, but this is only for tests and the tests seem to be happy with that change.

Reply:
yes, we do. the `create_unique_table_name` function generates the table name setting the table name length limit to 62. But, since we're appending the output of `create_unique_table_name` to the existing name, it might exceed 62 characters. 
Do we need to append the suffix to existing table name or can we directly replace it with 
`table.name = create_unique_table_name(UNIQUE_HASH_SIZE)` . Or if if we need the existing table name then how about do it like this: `table.name = create_unique_table_name(prefix=table.name, UNIQUE_HASH_SIZE)` and redefine the function to accept prefix and prepend it like we do in https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/table.py#L75

Reply:
Thanks @pankajkoti! I think you're right, we actually don't need to append.